lom/src/lib/lib_test/tx_test.go

```diff
@@ -1372,3 +1373,82 @@ func TestShutdown(t *testing.T) {
     //LogDebug("---------- COMPLETE ----------------")
 }
 
+/* Validates if configuration is returned from a valid json */
+func Test_GetIntConfigurationFromJson_ReturnsSuccessfully(t *testing.T) {
+	jsonString := "{\"abc\":123}"
+	result := GetIntConfigurationFromJson(jsonString, "abc", 5)
+	assert := assert.New(t)
+	assert.Equal(123, result)
+}
+
+/* Validates if default configuration is returned for empty json */
+func Test_GetIntConfigurationFromJson_ReturnsDefaultValueForEmptyJson(t *testing.T) {
+	jsonString := ""
+	result := GetIntConfigurationFromJson(jsonString, "abc", 5)
+	assert := assert.New(t)
+	assert.Equal(5, result)
+}
+
+/* Validates if default configuration is returned for invalid json */
+func Test_GetIntConfigurationFromJson_ReturnsDefaultValueForInvalidJson(t *testing.T) {
+	jsonString := "{\"abc\";abc}"
+	result := GetIntConfigurationFromJson(jsonString, "abc", 5)
+	assert := assert.New(t)
+	assert.Equal(5, result)
+}
+
+/* Validates if default value is returned when key not found */
+func Test_GetIntConfigurationFromJson_ReturnsDefaultValueForMissingConfigKey(t *testing.T) {
```

Comment:
Make it table driven

entry {
     in_String = "{\"abc\";abc}"
     result = 5
}
List of such entries

A single code that loops through

Reply:
Done.